### DISCUSSION TITLE
How do I change the default navigation keys?

### DISCUSSION THREAD

**Discussion by awulkan** [71/dance#273] [Q&A] [answered]
Hey, thanks for developing this cool extension.
I'm trying to figure out how to change the default keys for navigation from:
HJKL to JKLÖ (Nordic layout) so that I don't have to move my hand from the default position on the home row. But I don't really know how. Could anyone help?

**Comment by 71** [comment, marked as answer]
Dance commands are defined in VS Code keybindings, so you can just add new keybindings for these keys and remove the previous ones.

For instance, [these](https://github.com/71/dance/blob/ad61336bff165d29c5e674f3fd3916f7d6ba8bd7/package.json#L3513-L3536) are the default keybindings to go up. You can copy-paste that in `keybindings.json` and change the character that triggers the command. You will also have to delete the previous keybinding, e.g.

```json
{
  "key": "K",
  "when": "editorTextFocus && dance.mode == 'normal'",
  "command": "-dance.select.up.jump"
},
```

**Reply by awulkan** [reply]
Thank you. :)
